lib/diagnostic.rb

```diff
@@ -108,10 +109,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'Hash.new; person_hash = { favorite_number: 2, first_name: 'Angela'}'
```

Comment:
We were looking for the short hand syntax here:

`'person = {first_name: "Angela", favorite_number: 2}'`